packages/config/config/devices/0x0086/zw141.json

```diff
@@ -0,0 +1,229 @@
+// Aeotec ZW141
+// Nano Shutter
+{
+	"manufacturer": "Aeotec",
+	"manufacturerId": "0x0086",
+	"label": "ZW141",
+	"description": "Nano Shutter",
+	"devices": [
+		{
+			"productType": "0x0303",
+			"productId": "0x008d",
+			"zwaveAllianceId": 3132
+		},
+		{
+			"productType": "0x0903",
+			"productId": "0x008d",
+			"zwaveAllianceId": 3269
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"35": {
+			"label": "Moving Time From Up (left) to Down (right) For Curtain",
+			"description": "The available range for time set is 1 to 127 seconds",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 127,
+			"defaultValue": 127,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"80": {
+			"label": "Notification To Send to The Associated Devices (Group 1) on State Change",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Send nothing",
+					"value": 0
+				},
+				{
+					"label": "Send basic cc report",
+					"value": 1
+				}
+			]
+		},
+		"85": {
+			"label": "Operation Mode of External Switch",
+			"description": "Refer to the user manual for the description of operation mode 1 or 2",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
```

Comment:
Is the default really Operation Mode 2? I don't know what this or does, but it seems odd to me that 1 isn't the default.

Reply:
Looks like you're right based on the site linked in my previous comment. Updated 👍